docs/modules/ROOT/pages/configurations.adoc

```diff
@@ -16,6 +16,11 @@ The tools and configurations below are required to run aiSSEMBLE.
 * Continuous Deployment: https://docs.tilt.dev/install.html[Tilt,role=external,window=_blank] for local development
 * Source Code Management: https://git-scm.com/[git,role=external,window=_blank]
 
+==== Maven Build Cache
+aiSSEMBLE supports use of the Maven Build Cache to accelerate your project's development.  Newly created aiSSEMBLE
```

Comment:
A: I'd move the link to the  build cache docs to the words _Maven Build Cache_ here, and drop the line about existing projects entirely.  Existing projects will be informed of this via the release notes.